docker/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM scratch
+FROM alpine:3.3
 ADD mesos-dns_rootfs.tar.gz /
```

Comment:
This tarball would need to be adjusted then as well. Otherwise it will overwrite files like
```
/etc/group
/etc/hostname
/etc/hosts
/etc/nsswitch.conf
/etc/passwd
/etc/resolv.conf
/lib64/ld-linux-x86-64.so.2
/lib64/libc.so.6
/lib64/libnss_compat.so.2
/lib64/libnss_dns.so.2
/lib64/libnss_files.so.2
/lib64/libpthread.so.0
```
which I add to the rootfs tarball that's being generated by the docker-rootfs target. The assembly of the tarball is done in common/Makefile.